ocp_resources/resource.py

```diff
@@ -654,13 +655,21 @@ def update_replace(self, resource_dict):
         self.api.replace(body=resource_dict, name=self.name, namespace=self.namespace)
 
     @staticmethod
-    def _retry_cluster_exceptions(func):
+    def _retry_cluster_exceptions(
+        func,
+        **kwargs,
```

Comment:
Please replace with a parameter with a default; we should avoid using kwargs - 1. for usability 2. we know what we expect
so something like:
`cluster_exceptions=DEFAULT_CLUSTER_RETRY_EXCEPTIONS`

A user can then either not pass anything or pass their own (where they can decide which expections they's like to use)

Reply:
@rnetser this change was done based on @myakove 's recommendation on a previous review comment. I originally had something like what you recommended. Can you both please advice on how you want me to proceed here? https://github.com/RedHatQE/openshift-python-wrapper/pull/147#discussion_r701690506